components/user_settings/sidebar/user_settings_sidebar.jsx

```diff
@@ -290,10 +314,99 @@ export default class UserSettingsSidebar extends React.Component {
         );
     };
 
+    renderChannelSwitcherSection = () => {
+        let channelSwitcherSectionDescId = 'user.settings.sidebar.channelSwitcherSectionDesc.windows';
```

Comment:
When you rebase from master, be sure this string is wrapped inside the `t` function from `utils/i18n`.

Reply:
done